src/lib.rs

```diff
@@ -43,31 +40,35 @@ We need your help!
 
 */
 
+#![feature(test)]
```

Comment:
Hi Steve,

#![feature] can't be used in stable, are we ok with switching everything to using nightly?

Reply:
Hmmm... yes, you're right. I only put that in for benchmarking. I'll make that conditional, or remove it.

Reply:
I've moved the benchmarks into a separate folder. The code now compiles and runs under the stable toolchain.